src/store/getters.ts

```diff
@@ -752,6 +756,62 @@ export const getPriorityRequest = (state: StateIF): boolean => {
   return state.stateModel.newRequestModel.priorityRequest
 }
 
+export const getDesignationObject = (state: StateIF): any => {
+  if (getEntityTypeCd(state) && this.$designations[getEntityTypeCd(state)]) {
+    return this.$designations[getEntityTypeCd(state)]
+  }
+  return ''
```

Comment:
I think this should return null or an empty object instead.
(Normally this method will return an object, so when it fails ... ?)

Reply:
Good point, this will only every return `''` if there is no entity_type_cd selected or the `$designations` are unavailable.
That being said, this empty string here was probably there way of saying return false, since this method is used as a conditional in `getSplitNameDesignation` which in turn will handle that falsy value. 

Probably why they went with an empty string over an empty object, cuz i don't believe an empty object is falsy.

Reply:
You're right -- an empty object is not falsy. But null is :)